e2e/mocks/order-api/.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "es6": true,
+        "mocha": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "@typescript-eslint/no-unused-vars": "error",
+        "space-before-function-paren": "off",
+        "space-in-parens": "off",
+        "array-bracket-spacing": "off",
+        "semi": "off",
+        "@typescript-eslint/semi": ["error"]
+    },
+    "overrides": [{
+        "files": ["test/**"],
+        "rules": {
+            "no-unused-expressions": "off"
```

Comment:
This override is to allow chai expressions like `expect(something).to.exist` which the linter would normally disallow (which is why it is only applied on files under test directory).
Since chai is not used in `order-api`' we can optionally remove this